defold-spine/editor/src/spineext.clj

```diff
@@ -346,9 +346,23 @@
 (defn- set-constants! [^GL2 gl shader ro]
   (doall (map (fn [constant] (set-constant! gl shader constant)) (.m_Constants ro))))
 
+(defn- blend-factor-value-to-blend-mode [blend-factor-value]
+       (case blend-factor-value
+             0 :blend-mode-alpha
+             1 :blend-mode-add
+             2 :blend-mode-mult
+             3 :blend-mode-screen
+             :blend-mode-alpha))
+
 (defn- do-render-object! [^GL2 gl render-args shader renderable ro]
   (let [start (.m_VertexStart ro) ; the name is from the engine, but in this case refers to the index
         count (.m_VertexCount ro)
+        ro-blend-factor-value (.m_BlendFactor ro)
+        renderable-user-data (:user-data renderable)
+        renderable-blend-mode (:blend-mode renderable-user-data)
+        blend-mode (if (= renderable-blend-mode :blend-mode-inherit)
+                     (blend-factor-value-to-blend-mode ro-blend-factor-value)
```

Comment:
Personally, I would inline `(.m_BlendFactor ro)` here, since we don't need to access the field unless we have `:blend-mode-inherit`. The field access is slightly expensive since we don't have the type information and need to get the value through reflection.